modules/shared_vpc_access/outputs.tf

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+output "active_api_service_accounts" {
+  description = "List of active API service accounts in the service project."
+  value       = local.active_api_s_accounts
```

Comment:
I am wondering if this would cause issues with dependency ordering. My concern is that  `local.active_api_s_accounts` can be computed locally and doest not seem to dependent on any resources. Another thought is, we could output the project id and have a `depends_on` to enforce this.

Reply:
I do agree with you. Unfortunatelly `depends_on` block can not have variables in it. For example i can't do 
```hcl
output "something" {
  value = local.active_api_s_accounts
  depends_on = [var.project_id]
}```
The submodule doesn't need any outputs. I can just remove `outputs.tf`.

Reply:
That's not what a depends_on is meant to do - the point is to establish a connection with a resource.

Please add an output like this:
```
output "project_id" {
  value = var.project_id
  depends_on = [google_compute_subnetwork_iam_member.gke_shared_vpc_subnets, google_project_iam_member. gke_host_agent, google_project_iam_member. dataproc_shared_vpc_network_user]
}
```

Reply:
added